python/b2gedmntuples_cff.py

```diff
@@ -520,6 +520,26 @@
 #### FOR SYSTEMATICS
 jetVarsForSys = (
     cms.PSet(
+        tag = cms.untracked.string("jecUncertainty"),
+        quantity = cms.untracked.string("userFloat('jecUncertainty')")
+        ),
+    cms.PSet(
+        tag = cms.untracked.string("PtResolution"),
+        quantity = cms.untracked.string("userFloat('PtResolution')")
+        ),
+    cms.PSet(
+        tag = cms.untracked.string("ScaleFactor"),
+        quantity = cms.untracked.string("userFloat('ScaleFactor')")
+        ),
+    cms.PSet(
+        tag = cms.untracked.string("ScaleFactorUp"),
+        quantity = cms.untracked.string("userFloat('ScaleFactorUp')")
+        ),
+    cms.PSet(
+        tag = cms.untracked.string("ScaleFactorDown"),
+        quantity = cms.untracked.string("userFloat('ScaleFactorDown')")
+        ),
+    cms.PSet(
```

Comment:
@jkarancs sorry missed this PR, i made a comment inline about the tag names for the JER. The rest seems fine.